internal/pkg/mapper/variables/persist_test.go

```diff
@@ -44,3 +44,195 @@ func TestVariablesMapBeforePersist(t *testing.T) {
 		},
 	}, configManifest.Relations)
 }
+
+func TestVariablesValuesPersistFistRowIsDefault(t *testing.T) {
```

Comment:
Vysvetlim to najprv na testoch.

- pri operacii `persist` sa najde na disku novy adresar `variables`, eg: https://github.com/keboola/keboola-as-code/tree/main/internal/pkg/fixtures/local/persist-variables/main/extractor/ex-generic-v2/456-todos/variables
- v manifeste sa vytvori zaznam pre config komponenty `keboola.variables` + jednotlive rows (z adresara `values`) su hodnoty tychto premnnych
- to vsetko uz funguje, zostava posledna vec -> jeden z `rows` musia byt `default values`, na tie sa potom z konfiguracie komponenty, co pouziva `variables` odkazuje ako `variables_values_id`
- lenze pri operacii `persist` sa zatial ziadnemu `row` nepridavala relacia `VariablesValuesFor`, a to riesi tento PR